packages/cmf/src/cmfConnect.md

```diff
@@ -221,6 +221,32 @@ If you want to render some component conditionally, just pass "renderIf" prop (t
 You can also use Expression for this and customize this prop like "renderIfExpression" in
 CMF json configuration files
 
+## How to access state and mutate from the outside
```

Comment:
title may be misleading, the state is not mutated in sense of pointer value mutation, but it is updated
to avoid confusion i would rename this title to
`## Read and update component state from outside`